packages/xo-server/src/xapi/index.js

```diff
@@ -91,6 +91,14 @@ export const IPV6_CONFIG_MODES = ['None', 'DHCP', 'Static', 'Autoconf']
 
 // ===================================================================
 
+const XO_COMPRESS_VALUE_TO_XAPI = {
+  '': () => 'false',
+  native: () => 'true',
+  zstd: productBrand => (productBrand === 'XCP-ng' ? 'zstd' : 'true'),
+}
```

Comment:
Let's remove this for now and keep it simple.
If we send `'zstd'` but it's not supported, it will behave like `'false'`, which is good enough for now.

Please also create an issue for the next step: to add the detection and add a backup warning if not supported (but still do the backup).